internal/controller/monovertexrollout_controller.go

```diff
@@ -192,13 +192,13 @@ func (r *MonoVertexRolloutReconciler) reconcile(ctx context.Context, monoVertexR
 		Spec: monoVertexRollout.Spec.MonoVertex.Spec,
 	}
 
-	existingMonoVertexDef, err := kubernetes.GetCR(ctx, r.restConfig, newMonoVertexDef, "monovertices")
+	existingMonoVertexDef, err := kubernetes.GetLiveResource(ctx, newMonoVertexDef, "monovertices")
```

Comment:
oh, sorry. I guess the original issue said to just use the cache for getting `pipeline` and `isbsvc`, but actually I had just forgotten about `monovertex` - that is the same. And like `isbsvc`, I think we can just get it from the cache and not worry about getting a live version.